### ISSUE TITLE
Speed up asymmetry calculation

### ISSUE THREAD

**Issue by raquelalvarezbanos** [mantidproject/mantid#12158]
This issue was originally [TRAC 11319](http://trac.mantidproject.org/mantid/ticket/11319)

This ticket is blocked by : 
- [X] [TRAC9215](http://trac.mantidproject.org/mantid/ticket/9215)

PlotAsymmetryByLogValue loads a set of muon runs and computes the asymmetry as a function of some log value. Currently, the data loading and the asymmetry calculation are performed within the same loop. These tasks should be carried out separately so that users can play with asymmetry parameters (for instance integration limits) without having to load all the datasets every time.

**Comment by raquelalvarezbanos** [comment]
Not sure if this is still an issue. PABLV used to be too slow because every time it was run it would load and analyze the whole set of runs. But this problem was partially solved in #13157. Since that fix, when all the input properties stay the same between consecutive executions, only new runs are loaded and analyzed. The only potential issue remaining is that the algorithm will need to load & analyze everything when one of the inputs changes. In this case it would be useful to perform the analysis in parallel (this is what this issue is about). However, no one has complained about performance since #13157, so I'm marking this as Low Priority.
